src/api/internal.rs

```diff
@@ -1091,11 +1078,12 @@ impl<T: Pixel> ContextInner<T> {
         false,
         false,
       );
-      let rec = if frame_data.fi.show_frame {
-        Some(frame_data.fs.rec.clone())
+      let (rec, ref_f) = if frame_data.fi.show_frame {
```

Comment:
Commit d2d29da7357cb296f9d590ea97e16cd6b6ff0a7d, would be removed once #2186 has been landed